From 41d0e8d8cf2335a5b76139d3564fde8771f88efa Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:20 -0400
Subject: [redhat] kabi: Adding symbol num_physpages (mm/memory.c)

Message-id: <02963909a12d4ffdebc3778271447f94570a0d4d.1410510773.git.poros@redhat.com>
Patchwork-id: 95999
O-Subject: [PATCH RHEL7.1 140/233] kabi: Adding symbol num_physpages (mm/memory.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Used to get total no of physical pages in the system.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/num_physpages
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/num_physpages
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/num_physpages

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/num_physpages b/redhat/kabi/kabi-module/kabi_ppc64/num_physpages
new file mode 100644
index 0000000..8f0c25c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/num_physpages
@@ -0,0 +1,2 @@
+#1-
+0x0948cde9 num_physpages vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/num_physpages b/redhat/kabi/kabi-module/kabi_s390x/num_physpages
new file mode 100644
index 0000000..8f0c25c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/num_physpages
@@ -0,0 +1,2 @@
+#1-
+0x0948cde9 num_physpages vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/num_physpages b/redhat/kabi/kabi-module/kabi_x86_64/num_physpages
new file mode 100644
index 0000000..8f0c25c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/num_physpages
@@ -0,0 +1,2 @@
+#1-
+0x0948cde9 num_physpages vmlinux EXPORT_SYMBOL
-- 
1.7.1